include/flatbuffers/flatbuffers.h

```diff
@@ -2824,8 +2823,10 @@ inline const char * const *ElementaryTypeNames() {
 // GCC compilers.
 struct TypeCode {
   unsigned short base_type : 4;  // ElementaryType
-  unsigned short is_repeating : 1;  // Either vector (in table) or array (in struct)
```

Comment:
Can you move the comments to be above the item to avoid this weird formatting?

Reply:
Fixed